### DISCUSSION TITLE
Icon name reference

### DISCUSSION THREAD

**Discussion by wgong** [zauberzeug/nicegui#1764] [Show and tell]
NiceGui has `ui.icon` element that streamlit does NOT have.

For easy reference to Google Material Icon locally, I create a simple NiceGui app as a learning exercise

![image](https://github.com/zauberzeug/nicegui/assets/329928/bc6fe227-5744-4665-a261-e8fa90064eeb)

see [readme](https://github.com/wgong/py4kids/tree/master/lesson-16-gui/nicegui/examples#readme) for more details

**Comment by rodja** [comment]
Thats cool. But why have you not created the icon list in https://github.com/wgong/py4kids/blob/master/lesson-16-gui/nicegui/examples/icons/walk_tree_at_fixed_depth.ipynb? It could also be done directly in the NiceGUI application, I guess.

**Reply by wgong** [reply]
@rodja Thanks for your suggestion, my next step is to build a CRUD app using grid on icon data
